release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -287,6 +287,19 @@ With this release, {cgu-operator-first} uses a {rh-rhacm-first} feature to remed
 
 For more information, see xref:../edge_computing/cnf-talm-for-cluster-upgrades.adoc#talo-policies-concept_cnf-topology-aware-lifecycle-manager[Update policies on managed clusters].
 
+[id="ocp-4-16-edge-computing-accelerated-ztp_{context}"]
+==== Accelerated provisioning of {ztp} (Technology Preview)
+
+With this release, you can reduce the time taken for cluster installation by using accelerated provisioning of {ztp} for {sno}. 
+Accelerated ZTP speeds up installation by applying Day 2 manifests derived from policies at an earlier stage.
+
+The benefits of accelerated provisioning of {ztp} increase with the scale of your deployment. 
+Full acceleration gives more benefit on a larger number of clusters. 
+With a smaller number of clusters, the reduction in installation time is less significant.
+
+For more information, see xref:../edge_computing/ztp-deploying-far-edge-sites.html#ztp-sno-accelerated-ztp_ztp-deploying-far-edge-sites[Accelerated provisioning of {ztp}].
```

Comment:
The link still works in the preview, somehow, but our xrefs need to point to adoc files instead of html files:
```suggestion
For more information, see xref:../edge_computing/ztp-deploying-far-edge-sites.adoc#ztp-sno-accelerated-ztp_ztp-deploying-far-edge-sites[Accelerated provisioning of {ztp}].
```